### DISCUSSION TITLE
MAC系统23版的问题

### DISCUSSION THREAD

**Discussion by eric-cleath** [nextai-translator/nextai-translator#293] [General]
我的是mac系统，装了23版本：1、每次运行都弹出
删除重新添加也不行；2、快捷键调出的窗口是空白。
有什么解决办法呢？
![image](https://user-images.githubusercontent.com/60673256/225088201-998c4395-07a9-4750-b81c-1110e2b8f05c.png)

**Comment by yetone** [comment]
如果是 M1/M2 芯片，请按照 `aarch64` 版本，然后使用这种方法解决权限问题：https://github.com/yetone/openai-translator#troubleshooting

**Reply by eric-cleath** [reply]
谢谢。
我的机器是M2芯片，安装aarch64 版本，解决权限问题之后，可以正常使用。
但在多次操作之后，快捷键调出的窗口不能翻译，仍然显示上一次的内容。

只能退出程序，重新按介绍的方法解决权限问题才能正常使用。
